containers/README.md

```diff
@@ -0,0 +1,123 @@
+<!--
+    Copyright 2023 TII (SSRC) and the Ghaf contributors
+    SPDX-License-Identifier: CC-BY-SA-4.0
+-->
+
+# Containers
+
+Follow the instructions below to run various services inside Docker containers.
+
+## Hydra Container
+
+### Run Hydra in Container
+
+Custom configuration can be set with the `confs/hydra.local`.
+See the default configuration in the [confs/hydra.default](confs/hydra.default).
+
+[More detailed instructions to configure and run the Hydra instance](README.hydra)
+
+Here is an example for configuring Hydra to run on port 3001 and creating a store in the home directory:
+```
+HC_PORT=3001
+
+HC_BASE_LABEL="hydra-container"
+HC_PB_SRV="hydra"
+
+HC_STORE_PATH=/home/$USER/store
+
+PW_ADMIN="changeme"
+PW_AUTO="changeme"
+
+HC_CUSTOM_HOSTS=()
+```
+
+Build the Hydra image and create the store:
+```
+./build_hydra.sh
+./run_hydra.sh
+```
+The store gets initialized with the first run of container. When the first run have exited, you can start Hydra instance inside the container again.
```

Comment:
The store gets initialized with the first run of the container. When the first run exits, you can start the Hydra instance inside the container again.